website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,191 @@
+
+#### Better error readability for Hermes
+
+Hermes has added better error message when invoking an undefined callable.
+
+```
+    var x = undefined; x();
+    // Before: undefined is not a function
+    // After: x is not a function (it is undefined)
+```
+
+In addition, LogBox stack traces now filter out internal Hermes bytecode frames that are not relevant to app users.
+
+#### Improved error output of the React Native CLI
+
+0.72 ships with React Native CLI v11 which includes improvements to reduce duplication, clarify wording, reduce verbose stack traces, and add deep links to relevant docs in the following commands `init` , `run-android` and `run-ios`.
+
+You can find other improvements in the [React Native CLI changelogs](https://github.com/react-native-community/cli/releases).
+
+#### Faster Compilation and JSON Parsing in Hermes
+
+Hermes has improved the compilation time of large object literals. For example, in one reported issue, [#852](https://github.com/facebook/hermes/issues/852), a user had an entire dataset written out as a large object literal. By improving the de-duplication algorithm Hermes uses, compilation sped up by 97% ([221c](https://github.com/facebook/hermes/commit/221ce21a209e2e32a3eaaa2d9e28ca81842fad20)). These improvements will benefit build times for apps that bundle many objects.
+
+Multiple optimizations ([de9c](https://github.com/facebook/hermes/commit/de9cff2aa41fc1f297b568848143347823d73659), [6e2d](https://github.com/facebook/hermes/commit/6e2dd652c8d90c5d59737a81f66a259efffdcd00)) to JSON parsing have also landed, benefiting apps using libraries like redux-persist that rely heavily on JSON manipulation.
+
+#### More ECMAScript Support in Hermes
+
+Support for the following specifications in Hermes have landed in React Native 0.72:
+
+- `prototype.at` support for [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at), [TypedArray](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/at) and [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/at). See [#823](https://github.com/facebook/hermes/issues/823) ([ebe2](https://github.com/facebook/hermes/commit/ebe2915ac386a6b73dec39c2af4ac7063e68cd99)).
+- Implement [well-formed JSON.stringify](https://github.com/tc39/proposal-well-formed-stringify) ([d41d](https://github.com/facebook/hermes/commit/d41decf244aa814b1e58827a9de982f3b71667de)) to prevent ill-formed Unicode strings
+- Implement [AggregateError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError) ([9b25](https://github.com/facebook/hermes/commit/9b25a2530eb515f6c4fbd397ae290b6c97c049b2)) that represents several errors wrapped in a single error. Useful for multiple errors like from [`Promise.any()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/any) when all promises passed to it reject.
```

Comment:
Something that has bitten us before, if we advertise new Hermes features, we should make sure it works out of the box in TS.

In this case we should be good, because https://github.com/tsconfig/bases/commit/fd08789f6d296475dd4c130ed2a0259b2a9a8f60 added `es2021.promise` to our tsconfig.